config/tips.txt

```diff
@@ -1,92 +1,155 @@
-Coffee can keep you very warm. Drink a lot before running around through exposed space.
-Both cargo and the HoP can make you fill out a form for their services.
-You can destroy a blob by shooting emitters at its core.
-You can electrify a grille by having an exposed wire node underneath it.
-If you press the tab key, you can toggle hotkey mode, which allows you to move using WASD and perform other actions using hotkeys.
-You can hack MULE bots to let you ride them.
-You can hack most vending machines to obtain contraband items.
-You can use a multitool on the Cargo Ordering Console circuitboards to get contraband crates.
-The cryptographic sequencer (emag) has an enormous amount of functions beyond breaking open locks and hacking cyborgs, experiment!
-Touching anything without black, insulated, or brown gloves will leave fingerprints, which can be analyzed by detectives to find the suspect. Be careful of what you leave prints on!
-Blobs are extremely powerful and are impossible to defeat alone. Communicate with the crew and rally them against the threat!
-You can fight blob pieces diagonally, as they can only expand onto adjacent tiles in cardinal directions.
-Security huds and secglass huds can tell you if someone is implanted with a loyalty implant. Use this to your advantage in a revolution.
-Loyalty implants can only prevent someone from being turned into a cultist: unlike revolutionaries, it will not de-cult them if they have already been converted.
-Loyalty implants will break when used to de-convert a gangster. You will need to use two if you want to prevent them from being re-recruited.
-Don't neglect upgrading your machines with the machine parts produced by research! These can seriously improve the efficiency of your equipment.
-Cyborgs are impervious to fires and temperature, and can walk in a blazing inferno without worry. Don't shy away from setting the whole station on fire as a malfunctioning or rogue AI!
-The heads of staff, especially the Head of Security and the Captain, are usually extremely difficult to kill. If one of them is your target as a traitor, plan carefully.
-The Steal DNA sting from changelings counts for your genome absorb objective, but does not let you change your powers.
-Chemists are among the most powerful people on the station. They can survive almost anything thanks to all the chemicals in them that have basically replaced their blood.
-The station is capable of being entirely powered by the solar arrays, and is a much safer form of power over the singularity engine.
-The singularity engine is extremely dangerous if containment fails. Check back on it very frequently and call the shuttle immediately when, not if, it breaks free.
+Dragging yourself or someone else onto a chair or bed will buckle them to it, imobilizing them. If handcuffed, it will take time to resist out.

```

Comment:
You can't actually drag someone to a chair to buckle them to it unless they're on the same tile as the chair first. Probably should add that part.